src/Framework/BaseWebApplication.php

```diff
@@ -347,6 +347,11 @@ protected function configureTemplateEngine()
         if (isset($this['parameters']['userbase_url'])) {
             $this['twig']->addGlobal('userbase_url', $this['parameters']['userbase_url']);
         }
+        
+        $this['twig']->getExtension('core')->setDateFormat('d/m/Y', '%d days');
+        if(isset($this['parameters']['twig_date_format'])) {
+            $this['twig']->getExtension('core')->setDateFormat($this['parameters']['twig_date_format'], '%d days');
+        }
```

Comment:
@h-wang cc @bvklim, oh absolutely, let's make sure this functionality works nicely together.